tests/foreman/cli/test_acs.py

```diff
@@ -0,0 +1,277 @@
+"""Test class for Alternate Content Sources CLI
+
+:Requirement: AlternateContentSources
+
+:CaseAutomation: Automated
+
+:CaseLevel: Component
+
+:CaseComponent: AlternateContentSources
+
+:Team: Phoenix
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+import pytest
+from fauxfactory import gen_alphanumeric
+
+from robottelo.cli.base import CLIReturnCodeError
+from robottelo.cli.factory import make_product
+from robottelo.cli.factory import make_repository
```

Comment:
Use `target_sat.cli_factory.make_product` and `target_sat.cli_factory.make_repository` and remove these imports.